<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgAdjust</class>
 <widget class="QMainWindow" name="ImgAdjust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImgAdjust</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Adjust</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionCurve_Adjust"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveLUT"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionCurve_Adjust">
   <property name="text">
    <string>Curve Adjust</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
   <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveLUT">
   <property name="text">
    <string>SaveLUT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
